opencensus-monitoring/opentelemetry_config.yaml

```diff
@@ -0,0 +1,16 @@
+receivers:
+  vmimageage:
+    build_date: @BUILD_DATE@
+    vm_image_name: @IMAGE_NAME@
+
+exporters:
+  stackdriver/test_exporter:
+    project: imccarten-flex-test
+    metric_prefix: flex/opentelemetry
```

Comment:
No. If you check the code, opentelemetry writes Stackdriver metrics to custom.googleapis.com/opencensus by default, but if you include a domain in the metric prefix, it will be used. https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/59d068f8d8ff5b653916aa30cdc4e13c7f15d56e/metrics_proto_test.go#L1004 shows the tests for that behavior. (It's in the old open census codebase but opentelemetry collector makes calls to the same code)